packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3216,10 +3216,19 @@ export class ContainerRuntime
 			"Fluid.ContainerRuntime.Test.SummarizationRecoveryMethod",
 		);
 		if (recoveryMethod === "restart") {
-			this._summarizer?.stop("latestSummaryStateStale");
-			// TODO: stop execution, but don't cause a cascading series of errors.
 			const error = new GenericError("Restarting summarizer instead of refreshing");
-			this.closeFn(error);
+			this.mc.logger.sendTelemetryEvent(
+				{
+					...event,
+					eventName: "RestartInsteadOfRefreshFromServerFetch",
```

Comment:
```suggestion
					eventName: "ClosingSummarizerOnSummaryStale",
```

Closing seems more appropriate because restarting is not in control of this client.